src/subcommand/wallet/inscribe.rs

```diff
@@ -81,13 +83,38 @@ impl Inscribe {
       None => get_change_address(&client, &options)?,
     };
 
-    let (unsigned_commit_tx, reveal_tx, recovery_key_pair) =
+    let parent_location = if let Some(parent_id) = self.parent {
+      if let Some(satpoint) = index.get_inscription_satpoint_by_id(parent_id)? {
+        if !utxos.contains_key(&satpoint.outpoint) {
```

Comment:
Let's move the parent to a new change address, and get the value of the parent output from `utxos`, which will let us avoid the below lookups.